### PR TITLE
libretro: mouse, lightgun runahead polling

### PR DESCRIPTION
**Pull request by kps501**
no secondary core

### CODE REVIEW

**Comment by kps501** [comment]
Actually I think this needs checking by someone else.

File: libretro/libretro.cpp [hunk 1]
```diff
@@ -1310,7 +1330,18 @@ bool8 S9xDeinitUpdate(int width, int height)
 
 bool8 S9xContinueUpdate(int width, int height)
 {
-   return S9xDeinitUpdate(width, height);
+   if(interlace_speed==1)
+     return S9xDeinitUpdate(width, height);
+
+   else if(interlace_speed==2)
+     return true;
```
Comment (retro-wertz) [inline libretro/libretro.cpp]: why would one not want to render frames?

Comment (retro-wertz) [reply libretro/libretro.cpp]: No, what i mean is in interlace_speed=2 will just return true, and not pass to S9xDeinitUpdate() which renders the frame to screen...

Comment (kps501) [reply libretro/libretro.cpp]: Oh. Uhm. Idk honestly. If we S9xDeinitUpdate, then we will get the half-frame updates upstream doesn't want to show normally. Not sure what to do about this. :o

Comment (retro-wertz) [reply libretro/libretro.cpp]: RPMR seems to run fine even in interlaced mode. but this works for you then go ahead. for me its just unnecessary to have it as an option if only a few games if not only this game suffers from it...

imho

Comment (kps501) [reply libretro/libretro.cpp]: That's a good point! I like that thinking. The Chrono Trigger libretro hack doesn't need core option and quietly does something helpful. We can do that for Lufia 2 credits which is very very specific case.

Going to do that. Thanks! Shutting down PR and making clean one. Already goofed again with snes9x.h :(

**Pull request closed without merging**